wp-content/plugins/cpipr-lcdm/cpipr_lcdm.php

```diff
@@ -0,0 +1,314 @@
+<?php
+/*
+ * Plugin Name: CPI
+ * Description: Plugin that will create cpipr lcdm post types (video, infographics).
+ * Version: 1.0
+ * Author: Jorge Moreira 
+*/
+
+require_once dirname( __FILE__ ) . '/municipios_admin.php';
+
+/* Install municipios table */
+function lcdm_install_municipios_db_table () {
+    global $wpdb;
+
+    $table_name = $wpdb->prefix . 'municipios';
+
+    $charset_collate = $wpdb->get_charset_collate();
+
+    $sql = "CREATE TABLE $table_name (
+      id mediumint(9) NOT NULL AUTO_INCREMENT,
+      municipio varchar(60) NOT NULL,
+      tipo_asistencia varchar(60) NULL,
+      desastre varchar(80) NULL,
+      categoria varchar(80) NULL,
+      descripcion_categoria varchar(80) NULL,
+      total_obligado varchar(15) NULL,
+      fecha_obligacion date NULL,
+      total_desembolsado varchar(15) NULL,
+      total_pareo_fondos varchar(15) NULL,
+      fecha_ultimo_pago date NULL,
+      fecha_actualizacion date NULL,
+      PRIMARY KEY  (id)
+    ) $charset_collate;";
+
+    require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+    dbDelta( $sql );
+}
+register_activation_hook( __FILE__, 'lcdm_install_municipios_db_table' );
+
+
+add_action( 'init', 'create_video_cpt' );
+
+function get_video_cpt_supports() {
```

Comment:
Could you reuse this for both custom post types? It seems like this and `get_infographic_cpt_supports()` return the same array.